squad-server/plugins/discord-killfeed.js

```diff
@@ -0,0 +1,96 @@
+import DiscordBasePlugin from './discord-base-plugin.js';
+
+export default class DiscordKillFeed extends DiscordBasePlugin {
+  static get description() {
+    return (
+      'The <code>DiscordKillFeed</code> plugin logs all wounds and related information to a Discord channel for ' +
+      'admins to review.'
+    );
+  }
+
+  static get defaultEnabled() {
+    return true;
```

Comment:
I doubt many people will want to use this plugin, I would suggest setting the default to `false`.

Reply:
Done 👍